Formula/trezor-bridge.rb

```diff
@@ -0,0 +1,26 @@
+class TrezorBridge < Formula
+  desc "Trezor Communication Daemon"
+  homepage "https://github.com/trezor/trezord-go"
+  url "https://github.com/trezor/trezord-go/archive/refs/tags/v2.0.31.tar.gz"
+  sha256 "fd834a5bf04417cc50ed4a418d40de4c257cbc86edca01b07aa01a9cf818e60e"
+  license "LGPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build"
+    bin.install "trezord-go"
+  end
+
+  plist_options startup: true
+
+  service do
+    run opt_bin/"trezord-go"
+    keep_alive true
+    working_dir HOMEBREW_PREFIX
+  end
+
+  test do
+    system "trezord-go", "-version"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.